test_programs/execution_success/regression_8662/src/main.nr

```diff
@@ -0,0 +1,23 @@
+
+struct Context {
+    f: fn() -> bool
+}
+fn main(c: bool) -> pub bool {
```

Comment:
Missing a stdout?

Reply:
You mean the `stdout.txt` for the test? I thought they were replaced by insta in https://github.com/noir-lang/noir/pull/8576

Reply:
oops yes